src/components/tabController/TabBar.tsx

```diff
@@ -117,7 +117,7 @@ interface Props extends TabControllerBarProps, BaseComponentInjectedProps, Forwa
   children?: ChildProps[] | ChildProps;
 }
 
-const FADER_PROPS = {size: 76, tintColor: Colors.$backgroundDefault};
+const FADER_PROPS = {size: 76, tintColor: Colors.$backgroundElevated};
```

Comment:
Are you sure about it?
I used the `backgroundElevated` originally and you said it looks better with `backgroundDefault`

Reply:
It'll be `$backgroundElevated` but will be handled within the `Fader`

Reply:
Are you sure that the default fader color should be `$backgroundElevated` and not `$backgroundDefault`? 
maybe it should be `$backgroundDefault` and only here we should override it with `$backgroundElevated`..

Reply:
That's what Yulia said in the relevant channel (can't link here).
In any case, if there will be errors we can fix them later.

Reply:
Changed